csharp/device/iothub_csharp_client.sln

```diff
@@ -28,6 +28,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NuGet", "NuGet", "{84052DC5
 		NuGet\Microsoft.Azure.Devices.Client.nuspec = NuGet\Microsoft.Azure.Devices.Client.nuspec
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Azure.Devices.Client.Http.NetMF_44", "Microsoft.Azure.Devices.Client.NetMF\HttpClient_44\Microsoft.Azure.Devices.Client.Http.NetMF_44.csproj", "{1B0C95B7-D1C7-4283-A495-044803FD034A}"
```

Comment:
Adding the .NET MF projects in the existing solution will break our build and force an additional dependency on the solution (the .NET MF SDK). I would suggest we use a separate solution for the .NET MicroFramework.